### PR TITLE
Feature: edit task

### PR DESCRIPTION
**Pull request by rhlc**

### CODE REVIEW

**Comment by GermaVinsmoke** [comment]
This is failing to build. Typescript error.

File: src/components/AcceptTask/FileAttachmetCard.tsx [hunk 1]
```diff
@@ -4,17 +4,26 @@ import styles from './index.module.scss';
 
 interface IFileAttachmentCard {
   label: string;
+  skills?: string;
 }
 
-const FileAttachmentCard: FC<IFileAttachmentCard> = ({ label }) => {
+const FileAttachmentCard: FC<IFileAttachmentCard> = ({ label, skills }) => {
```
Comment (GermaVinsmoke) [inline src/components/AcceptTask/FileAttachmetCard.tsx:10]: What's skills doing in this component?

File: src/components/AcceptTask/TaskDetail.tsx [hunk 2]
```diff
@@ -91,13 +93,20 @@ const TaskDetail: FC<ITaskDetail> = ({
     return null;
   };
 
+  const reformatDate = (dateStr: string) => {
+    const dArr = dateStr?.split('-');
+    if (dArr) return `${dArr[2]}/${dArr[1]}/${dArr[0]?.substring(0)}`;
+    return null;
+  };
```
Comment (GermaVinsmoke) [inline src/components/AcceptTask/TaskDetail.tsx]: `new Date("2022-07-30").toLocaleDateString("en-GB")`

File: src/components/AcceptTask/TaskDetail.tsx [hunk 3]
```diff
@@ -148,7 +157,20 @@ const TaskDetail: FC<ITaskDetail> = ({
           </div>
         )}
       </div>
-
+      <div className={styles['project-description']}>
+        <p> Skills:</p>
+        {selectedTask?.taskSkills?.length > 0 && (
+          <div className={styles['project-attachments']}>
+            {selectedTask?.taskSkills?.map((taskSkills: any, index: number) => (
+              <FileAttachmentCard
```
Comment (GermaVinsmoke) [inline src/components/AcceptTask/TaskDetail.tsx]: Why are we using `<FileAttachmentCard />` for rendering skills data?

File: src/components/AcceptTask/FileAttachmetCard.tsx (hunk 1 above)
Comment (rhlc) [reply src/components/AcceptTask/FileAttachmetCard.tsx:10]: I didn't want to use separate component for skills. This way, we can show skills and attachments using the same component.

File: src/components/AcceptTask/TaskDetail.tsx (hunk 3 above)
Comment (rhlc) [reply src/components/AcceptTask/TaskDetail.tsx]: Yes

File: src/components/AcceptTask/FileAttachmetCard.tsx (hunk 1 above)
Comment (GermaVinsmoke) [reply src/components/AcceptTask/FileAttachmetCard.tsx:10]: Okay but no one will ever know that this component can render skills also

File: src/config/index.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@ interface Environment {
 
 const configs: Environment = {
   local: {
-    ApiBaseUrl: 'http://localhost:3001/local',
+    ApiBaseUrl: 'http://localhost:3001',
```
Comment (GermaVinsmoke) [inline src/config/index.ts]: You removed this.

**Review by GermaVinsmoke — CHANGES_REQUESTED** [review]

**Review by GermaVinsmoke — APPROVED** [review]

**Comment by GermaVinsmoke** [comment]
The build is failing for both the Pull requests.

**Pull request merged**